stdlib/2/pydoc.pyi

```diff
@@ -0,0 +1,176 @@
+    def repr_str(self, x: str, level: complex) -> str: ...
+    def repr_instance(self, x: object, level: complex) -> str: ...
+
+class TextDoc(Doc):
+    def repr(self, object: object) -> str: ...
+    def bold(self, text: str) -> str: ...
+    def indent(self, text: str, prefix: str = ...) -> str: ...
+    def section(self, title: str, contents: str) -> str: ...
+    def formattree(self, tree: List[Union[Tuple[type, Tuple[type, ...]], list]], modname: str, parent: Optional[type] = ..., prefix: str = ...) -> str: ...
+    def docmodule(self, object: object, name: Optional[str] = ..., mod: Optional[Any] = ..., *ignored) -> str: ...
+    def docclass(self, object: object, name: Optional[str] = ..., mod: Optional[str] = ..., *ignored) -> str: ...
+    def formatvalue(self, object: object) -> str: ...
+    def docroutine(self, object: object, name: Optional[str] = ..., mod: Optional[str] = ..., cl: Optional[Any] = ..., *ignored) -> str: ...
+    def docproperty(self, object: object, name: Optional[str] = ..., mod: Optional[Any] = ..., cl: Optional[Any] = ..., *ignored) -> str: ...
+    def docdata(self, object: object, name: Optional[str] = ..., mod: Optional[str] = ..., cl: Optional[Any] = ..., *ignored) -> str: ...
+    def docother(self, object: object, name: Optional[str] = ..., mod: Optional[str] = ..., parent: Optional[str] = ..., maxlen: Optional[int] = ..., doc: Optional[Any] = ..., *ignored) -> str: ...
+
+def pager(text: str) -> None: ...
+def getpager() -> Callable[[str], str]: ...
+def plain(text: str) -> str: ...
+def pipepager(text: str, cmd: str) -> str: ...
+def tempfilepager(text: str, cmd: str) -> str: ...
+def ttypager(text: str) -> str: ...
+def plainpager(text: str) -> str: ...
+def describe(thing: Any) -> str: ...
+def locate(path: str, forceload: bool = ...) -> object: ...
+
+text = ...  # type: TextDoc
+html = ...  # type: HTMLDoc
+
+class _OldStyleClass: ...
+
+def resolve(thing: Union[str, object], forceload: bool = ...) -> Optional[Tuple[object, str]]: ...
+def render_doc(thing: Union[str, object], title: str = ..., forceload: bool = ...) -> str: ...
+def doc(thing: Union[str, object], title: str = ..., forceload: bool = ...) -> str: ...
+def writedoc(thing: Union[str, object], forceload: bool = ...) -> None: ...
+def writedocs(dir: str, pkgpath: str = ..., done: Optional[Any] = ...) -> None: ...
+
+class Helper:
+    keywords = ...  # type: MutableMapping[str, Union[str, Tuple[str, str]]]
+    symbols = ...  # type: MutableMapping[str, str]
+    topics = ...  # type: MutableMapping[str, Union[str, Tuple[str, ...]]]
+    def __init__(self, input: Optional[IO[str]] = ..., output: Optional[IO[str]] = ...) -> None: ...
+    input = ...  # type: IO[str]
+    output = ...  # type: IO[str]
+    def __call__(self, request: Union[str, Helper, object] = ...) -> None: ...
+    def interact(self) -> None: ...
+    def getline(self, prompt: str) -> str: ...
+    def help(self, request: Union[str, Helper]) -> None: ...
```

Comment:
I think request is actually just Any; there's an else block covering anything not a str or Helper.